jetstream/external_config.py

```diff
@@ -37,32 +37,29 @@ class ExternalConfigCollection:
 
     configs: List[ExternalConfig] = attr.Factory(list)
 
-    JETSTREAM_CONFIG_REPO = "mozilla/jetstream-config"
+    JETSTREAM_CONFIG_URL = "https://github.com/mozilla/jetstream-config"
 
     @classmethod
     def from_github_repo(cls) -> "ExternalConfigCollection":
         """Pull in external config files."""
+        # download files to tmp directory
+        tmp_dir = Path(tempfile.mkdtemp())
+        repo = Repo.clone_from(cls.JETSTREAM_CONFIG_URL, tmp_dir)
```

Comment:
If we're gonna do this a couple times, can we wrap this in a class that we can test?

I think we should also try to clean up the temporary repo after we're done.

The context manager pattern / contextlib.contextmanager decorator could help: https://docs.python.org/3/library/contextlib.html#contextlib.contextmanager